packages/babel-node/test/index.js

```diff
@@ -112,7 +112,7 @@ const buildTest = function(binName, testName, opts) {
 
     args = args.concat(opts.args);
 
-    const spawn = child.spawn(process.execPath, args);
+    const spawn = child.spawn("node", args);
```

Comment:
What if `node` is not resolvable from `PATH` env? i.e. User can invoke babel-node via absolute path of `node` bin.

Reply:
I think I changed this because `process.execPath` was not working, let me check again.